docs/configuration.md

```diff
@@ -349,6 +349,23 @@ of the most common options to set are:
   </td>
   <td>3.0.0</td>
 </tr>
+<tr>
+  <td><code>spark.executor.allowSparkContext</code></td>
```

Comment:
https://github.com/apache/spark/blob/22483167e20208e40e24abe6898b2102ddaf4fc9/core/src/main/scala/org/apache/spark/internal/config/package.scala#L2206-L2211

Reply:
BTW, I guess we don't want to expose this, @dcoliversun . There is no good for users.

Reply:
OK. Better to mark as internal configuration? If so, I will make new PR to solve it.